config/routes.rb

```diff
@@ -2,6 +2,7 @@
   devise_for :users
   resources :pets do
     resources :rents, only: [:new, :create]
+    resources :reviews, only: [:create]
```

Comment:
just a question: you only nest the create of reviews (not new?)

Reply:
We don't have a separate view for new